packages/core/src/Stores/traders-hub-store.js

```diff
@@ -20,7 +21,7 @@ export default class TradersHubStore extends BaseStore {
     is_account_type_modal_visible = false;
     account_type_card = '';
     selected_platform_type = 'options';
-    active_index = 0;
+    active_indx = 0;
```

Comment:
as I chcked there is no use of `active_indx` and `active_index`, please recheck and remove it